frontend/src/main/scala/akkaviz/frontend/components/ThroughputGraphTab.scala

```diff
@@ -59,28 +68,44 @@ class ThroughputGraphViewTab(implicit ctx: Ctx.Owner) extends Tab with FancyColo
     items.add(item)
   }
 
+  def colorForActor(ref: String): Color = {
+    colorForString(ref, 0.2, 0.2)
+  }
+
   groups.on("add", { (event: String, p: Properties[Group], sender: String | Double) =>
-    groupVisibility() = groupVisibility.now ++ (p.items.map(_ -> true))
+    groupVisibility() = groupVisibility.now ++ p.items.map(_ -> false)
   })
 
   private[this] def removeOldItems(): Unit = {
     val range = graph.getWindow()
     val interval = range.end.valueOf() - range.start.valueOf()
 
-    val oldIds = items.getIds(js.Dynamic.literal(filter = { (item: Item) =>
+    val oldIds = items.getIds(literal(filter = { (item: Item) =>
       item.x.valueOf() < (range.start.valueOf() - interval)
     }))
     items.remove(oldIds)
   }
 
-  private[this] val groupUpdate = groupVisibility.triggerLater {
-    val g = groupVisibility.now
-    val options = js.Dynamic.literal(groups = js.Dynamic.literal(
+  private[this] val groupUpdate = groupVisibility.foreach { g =>
+    val options = literal(groups = literal(
       visibility = js.Dictionary[Boolean](g.toSeq: _*)
     ))
     console.log(options)
     graph.setOptions(options)
   }
 
-  override def onCreate(): Unit = ()
+  override def onCreate(): Unit = {
+    window.requestAnimationFrame(autoScroll _)
+  }
+
+  def autoScroll(d: Double): Unit = {
+    val graphWindow = graph.getWindow()
+    val interval = graphWindow.end.valueOf() - graphWindow.start.valueOf()
+    val now = new Date().valueOf()
+    val start = new Date(now - interval)
+    val end = new Date(now)
+
+    graph.setWindow(start, end, literal(animation = false))
+    window.requestAnimationFrame(autoScroll _)
```

Comment:
it's what they actually recommend in the examples (see source)
http://visjs.org/examples/graph2d/15_streaming_data.html